packages/zowe-explorer/src/tools/ZoweTerminal.ts

```diff
@@ -19,45 +19,34 @@ export class ZoweTerminal implements vscode.Pseudoterminal {
         EMPTY_LINE: `${this.mTermX} `,
         CLEAR_ALL: "\x1b[2J\x1b[3J\x1b[;H",
         CLEAR_LINE: `\x1b[2K\r`,
-        CTRL_C: "\x03",
         END: "\x1b[F",
         HOME: "\x1b[H",
         CMD_LEFT: "\x01", // MacOS HOME
+        CTRL_C: "\x03",
+        CTRL_D: "\x04",
         CMD_RIGHT: "\x05", // MacOS END
+        CTRL_BACKSPACE: "\x08",
+        TAB: "\x09",
+        CMD_BACKSPACE: "\x15",
+        OPT_BACKSPACE: "\x17",
+        OPT_CMD_BACKSPACE: "\x1b\x7F",
+        BACKSPACE: "\x7f",
+        INSERT: "\x1b[2~",
         DEL: "\x1b[3~",
+        PAGE_UP: "\x1b[5~",
+        PAGE_DOWN: "\x1b[6~",
```

Comment:
Thanks for adding these 🙏

Reply:
Thank you for testing! 
Some of those where a bit strange to figure out with a Mac 😋